src/components/CampsiteListItem.js

```diff
@@ -75,25 +74,26 @@ const CampsiteListItem = ({site, getSiteDetail, navigate}) => {
 };
 
 const styles = StyleSheet.create({
+    titleContainerStyle:{
+        paddingLeft: 65
+    },
     IconContainer: {
         flexDirection: 'row',
-        paddingLeft: 10,
+        paddingLeft: 75,
         paddingTop: 5
     },
     badgeWrapperStyle: {
-        width: 100,
         marginLeft: 10
     },
     badgeContainerStyle: {
         backgroundColor: blueGreenNav
     },
-    siteAvatarStyle: {
-        flex: 1,
-        width: 120
-    },
     siteAvatarContainerStyle: {
+        marginLeft: 50,
+        borderRadius: 0,
+        paddingLeft: 0,
         height: 150,
-        width: 120
+        width: 150
```

Comment:
Had to do a bunch of style changes to get the list items working again after the RNE upgrade. Feel free to continue to tweak.